cassandra-driver/cassandra-driver.d.ts

```diff
@@ -375,8 +375,9 @@ declare module "cassandra-driver" {
     }
 
     interface Row {
-      get(columnName: string|number): { [key:string]:any; };
-      values(): Array<{ [key:string]:any; }>;
+      get(columnName: string|number):any;
```

Comment:
Why switch to `any` here?  Can you add a test case that would not work with a string-indexed object?